modules/021-cni-cilium/templates/operator/rbac-for-us.yaml

```diff
@@ -67,6 +67,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - list
+  - watch
```

Comment:
Why are these rights not needed in previous versions of the cilium agent?

Reply:
It was added in v1.11.5
https://github.com/cilium/cilium/commit/3369db77bf3d13f38709a5253c8badd05b85e342